samples/powershell/AddSharePointMappings.ps1

```diff
@@ -1,16 +1,20 @@
 Try {
-    [System.Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
     [System.Net.HttpWebRequest]::DefaultMaximumErrorResponseLength = -1
     Import-Module ((Split-Path -Parent $MyInvocation.MyCommand.Definition) + "\Common.ps1") -Force
     Import-Module -Name "Fly.Client"
+    #Get the global configuration object to set Fly_API_Endpoint and your access token
     $Configuration = Get-Configuration
     $Configuration["BaseUrl"] = "{Fly_API_Endpoint}"
     $Configuration.AccessToken = "YOUR_BEARER_TOKEN"
+    #Specify the file path of the project mappings to import, only support csv format
     $FilePath = 'C:\Data\50 mapping SP.csv'
```

Comment:
A comment should place to describe the csv format？

Reply:
Yes, we can remind customers to refer to the csv format when importing mapping in Fly UI